Dockerfile

```diff
@@ -4,6 +4,7 @@ LABEL com.connexta.application.name=search
 ARG JAR_FILE
 COPY ${JAR_FILE} /search
 ENTRYPOINT ["/search"]
+RUN chmod 755 /search
 EXPOSE 8080 10042 10052
```

Comment:
Not that removing it should be part of this ticket, but are we using any of the JMX stuff? https://github.com/connexta/ion-search/pull/75/files#diff-3254677a7917c6c01f55212f86c57fbfR12-R18

Reply:
I think so, for live debugging.